server/hotrod/src/main/java/org/infinispan/server/hotrod/configuration/Attribute.java

```diff
@@ -16,6 +16,7 @@ public enum Attribute {
    HOST_NAME,
    LAZY_RETRIEVAL,
    LOCK_TIMEOUT,
+   MAX_CONTENT_LENGTH,
```

Comment:
`MAX_CONTENT_LENGTH` should be in `org.infinispan.server.core.configuration.Attribute`. Remove the one in `org.infinispan.server.configuration.rest.Attribute`.